src/main.rs

```diff
@@ -219,6 +223,30 @@ async fn wrapped_main(config: Arc<Configuration>) -> Result<()> {
         PROGRESS_BAR.join().unwrap();
     });
 
+    // check if update_app is true
+    if config.update_app {
+        let target_os = format!("{}-{}", ARCH, OS);
+        tokio::task::spawn_blocking(move || {
+            let status = self_update::backends::github::Update::configure()
+                .repo_owner("epi052")
+                .repo_name("feroxbuster")
+                .bin_name("feroxbuster")
+                .target(target_os.as_str())
+                .show_download_progress(true)
+                .current_version(cargo_crate_version!())
+                .build()
+                .unwrap()
+                .update()
+                .unwrap();
```

Comment:
this can fail due to network issues, when the user enters n, and a bunch of others. we need to handle the error cases instead of unwrapping